docs/guidance/design/user_needs.md

```diff
@@ -4,47 +4,64 @@ Open data is a means, not an end in itself. The first stage of any work with OCD
+The private sector can use open contracting data to understand the potential pipeline of procurement opportunities and identify the most competitive markets.  A government agency can use open contracting data to understand trends in past purchases to better plan for future procedures, analyze particular sectors to evaluate competition and identify  potential market concentration.  Moreover, authorities can use contracting data to understand their supplier base in order to design and implement strategies to increase vendor diversity, such as [increasing the participation of women owned businesses](https://www.open-contracting.org/what-is-open-contracting/gender/), small and medium enterprises, minority owned firms or firms committed to sustainability.  
 
-These users want to analyze trends in prices and supplier performance, including in terms of quality and duration.
+**Key data fields**: Information about the procurement method used, the awarded suppliers, the companies that submit the bids, the tender and award values, details on the items procured are key fields to calculate relevant indicators related to market opportunity.  
 
-Comparable data using common codelists and the availability of unit prices are particularly important for value for money use cases.
 
-## Detecting fraud and corruption
+## Value for money in procurement
+
+Value for money refers to the effective, efficient and economic use of resources in public procurement across the different stages of the process. This means value for money might not be achieved considering only the price, but also assessing other non-price attributes such as quality of the items purchased and the efficiency of the process.  
+ 
+In public procurement, for instance, value for money can be achieved when a contract is implemented competently (in a quality manner and in accordance with specifications) in a timely manner (achieving specified milestones by the specified dates) for a competitive price (at or below estimate).
 
-All stakeholders (civil society, the private sector, government and donors) have an interest in identifying and combating corruption in public contracting. Open contracting data can be used to scrutinize procurement documents and data for ‘red flags’ that might indicate public monies are being mis-used.
+Open contracting data can help public officials to get good value for money on the goods and services in the procurement process, and can also help in identifying whether value for money has been achieved in concluded contracts. Users from the private sector can analyze trends in prices and supplier performance, including terms of quality and duration, in order to identify better opportunities.  Civil society (e.g. NGOs, journalists, community organizations, etc.) can analyze this data to monitor government spending.
```

Comment:
"monitor government spending" isn't clearly associated with value for money (it can be any of the use cases). I changed this to "monitor government overspending" so it's more clearly relevant to value for money. Otherwise the sentence is just saying "Civil society can also analyze value for money."